elements/cp-themeset/sass/_colors.scss

```diff
@@ -22,6 +22,7 @@
   //  THEME BASE PALETTE COLORS
   // ========================================================================
   */
+  --rhe-theme--color--transparent:          transparent;
```

Comment:
I wonder if it makes sense to have colors defined as CSS variables at all.  It seems like colors as Sass variables is sufficient and won't add to the compiled CSS.